src/app/session-page-test/session-page-test.component.html

```diff
@@ -4,4 +4,9 @@
   <input #name>
   <button (click)="addStudentToDatabase(name.value)">Log In</button>
   <p>UserID: {{ (this.authService.currentUser$ | async)?.uid }}</p>
+
+  <button (click)="getStudentsFromDatabase()">See all students</button>
+  <div *ngFor="let student of studentList">
```

Comment:
It doesn't really matter here in this test component, but in the future, you may want to consider making `studentList` an observable. Then you could write:

```html
  <div *ngFor="let student of studentList$ | async">
```

This has the benefit that studentList$ will unsubscribe automatically when you navigate to a different page. (The way the test component is set up right now, the subscriptions never go away. :upside_down_face:)

Reply:
Oh, it looks like that's what you're doing in `session-lobby.component.ts` :) Looks good!